src/Layout/LayoutColumn/index.js

```diff
@@ -1,14 +1,27 @@
 // @flow
 import * as React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 import defaultTheme from "../../defaultTheme";
 import getViewportHideStyles from "../../Hide/helpers/getViewportHideStyles";
+import { StyledCard } from "../../Card";
+import mq from "../../utils/mediaQuery";
 
 import type { Props } from "./index";
 
 const StyledColumn = styled.div`
   ${({ hideOn }) => !!hideOn && getViewportHideStyles(hideOn)};
+
+  ${StyledCard} {
+    margin: 0 -${({ theme }) => theme.orbit.spaceMedium};
```

Comment:
My recommendation is to set strictly what's necessary, with this shorthand you can accidentally override a possible `margin-top` or `margin-bottom` down the road.

```suggestion
    margin-left: -${({ theme }) => theme.orbit.spaceMedium};
    margin-right: -${({ theme }) => theme.orbit.spaceMedium};
```

Reply:
You're right! Especially if more Cards would be stacked with `Stack` component.
We can also don't have to solve RTL with functions that we have. It is set up for both horizontal sides, so it would be redundant.